src/native/managed/cdacreader/src/Contracts/RuntimeTypeSystem_1.cs

```diff
@@ -698,14 +698,15 @@ public bool IsArrayMethod(MethodDescHandle methodDescHandle, out ArrayFunctionTy
             return false;
         }
 
-        int arrayMethodIndex = methodDesc.Slot - GetNumVtableSlots(GetTypeHandle(methodDesc.MethodTable));
+        MethodTable methodTable = _methodTables[GetTypeHandle(methodDesc.MethodTable).Address];
```

Comment:
Does this warrant a comment? I really don't know how much this is trying to capture "the design" of the runtime vs current implementation detail. If I saw this I'd be interested in where I could confirm it in the runtime, but that could be obvious and I'm ignorant.